connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -174,7 +174,14 @@ private class UCProxy extends TableCatalog {
 
   override def alterTable(ident: Identifier, changes: TableChange*): Table = ???
 
-  override def dropTable(ident: Identifier): Boolean = ???
+  override def dropTable(ident: Identifier): Boolean = {
+    if (ident.namespace().length > 1) {
```

Comment:
can we add a common method to do namespace check?

Reply:
ah right. done